### ISSUE TITLE
documentation correction for reporter: email

### ISSUE THREAD

**Issue by jprokos** [mborsetti/webchanges#34]
An example that tripped me up and should be edited. I would "contribute" but fear I would do more harm.

The example as shown is missing the email sub-directive `smtp`. It's correct in the default config.py file.

```
report:
  tz: null
  email:
    enabled: true  # don't forget to set this to true! :)
    from: 'your.username@gmail.com'  # (edit accordingly; don't use your primary account for this!!)
    to: 'your.destination@example.org'  # The email address of where want to receive reports
    subject: '[webchanges] {count} changes: {jobs}'
    html: true
    method: 'smtp'
      host: 'smtp.gmail.com'
      user: 'your.username@gmail.com'  # (edit accordingly; don't use your primary account for this!!)
      port: 587
      starttls: true
      auth: true
      insecure_password: 'this_is_my_secret_password'
```

it should look like this:
```
report:
  tz: null
  email:
    enabled: true  # don't forget to set this to true! :)
    from: 'your.username@gmail.com'  # (edit accordingly; don't use your primary account for this!!)
    to: 'your.destination@example.org'  # The email address of where want to receive reports
    subject: '[webchanges] {count} changes: {jobs}'
    html: true
    method: 'smtp'
    smtp:
      host: 'smtp.gmail.com'
      user: 'your.username@gmail.com'  # (edit accordingly; don't use your primary account for this!!)
      port: 587
      starttls: true
      auth: true
      insecure_password: 'this_is_my_secret_password'
```

**Comment by mborsetti** [comment]
Thanks! No worries about contributing with a PR; easy fix with huge consequences!

**Comment by mborsetti** [comment]
Fixed in 3.9.2
